flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/SubtaskExecutionAttemptDetailsInfo.java

```diff
@@ -261,6 +261,53 @@ public static SubtaskExecutionAttemptDetailsInfo create(
                 duration,
                 ioMetricsInfo,
                 taskmanagerId,
-                getExecutionStateDuration(execution));
+                getExecutionStateDuration(execution),
+                otherConcurrentAttempts);
+    }
+
+    public List<SubtaskExecutionAttemptDetailsInfo> getOtherConcurrentAttempts() {
```

Comment:
This method seems not called?

Reply:
Quite a few getters in this class are not called. I'm not sure if we should add the getter or not. I'll move this method next to other getters first. Please let me know if it should actually be removed.